### PR TITLE
Fix error message showed when opening XLS file

### PR DESCRIPTION
**Pull request by MarwenBENALAYA**
I am targeting this branch, because {When downloading XLS file and opening it, a popup with message "File format and extension of 'filename.xls' don't match. The file could be corrupted or unsafe. Unless you trust its source, don't open it. Do you want to open it anyway?" showed. because the content file is HTML.}.

In case of bug fix, `1.x` **MUST** be targeted.

## Changelog

```markdown

### Changed
- Changing methods (XlsWriter::open(), XlsWriter::close(), XlsWriter::write($data), XlsWriter::init($data)) and using PHPExcel to cretae content of XLS file
```

## To do

- [ ] Update the tests

## Subject

- After downloading an XLS file, when I want to opening this file with Microsoft office Excel an message dialog showed "File format and extension of 'filename.xls' don't match. The file could be corrupted or unsafe. Unless you trust its source, don't open it. Do you want to open it anyway?" because the content of the file is HTML, I'm using a PHPExcel to generate a file

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,12 +21,13 @@
         "matthiasnoback/symfony-dependency-injection-test": "^1.0",
         "propel/propel1": "^1.6",
         "sllh/php-cs-fixer-styleci-bridge": "^2.0",
-        "symfony/dependency-injection": "^2.8 || ^3.2",
-        "symfony/http-foundation": "^2.8 || ^3.2",
-        "symfony/http-kernel": "^2.8 || ^3.2",
-        "symfony/phpunit-bridge": "^3.3",
-        "symfony/property-access": "^2.8 || ^3.2",
-        "symfony/routing": "^2.8 || ^3.2"
+        "symfony/dependency-injection": "^2.3 || ^3.0",
```
Comment (core23) [inline composer.json]: Please don't lower the deps

**Comment by OskarStark** [comment]
@MarwenBENALAYA please work in PR #166 and read my comment: https://github.com/sonata-project/exporter/pull/166#issuecomment-340071971

**Comment by Soullivaneuh** [comment]
Erf, did the same on the other PR. :confused: 

I think @MarwenBENALAYA wanted to use a specifiv branch for this PR.

Let's keep this one for this now.

**Comment by Soullivaneuh** [comment]
@OskarStark https://github.com/sonata-project/exporter/pull/166#issuecomment-340071971

I really don't know right now, I don't use this feature.

**Comment by OskarStark** [comment]
> Let's keep this one for this now.

OK, but for the future we should keep the first one created, because there could be some comments

**Comment by OskarStark** [comment]
If you want to finish this PR please reopen.

Thanks you!

**Pull request closed without merging**